tensorflow/lite/micro/tools/make/targets/disco_f746ng_makefile.inc

```diff
@@ -0,0 +1,10 @@
+# Settings for mbed platforms.
+ifeq ($(TARGET), disco_f746ng)
```

Comment:
these two lines can be removed since we only include the target-specific makefile:
https://github.com/tensorflow/tensorflow/blob/f8b99a48af0fde7b450c9630c825cebdfbba7c6a/tensorflow/lite/micro/tools/make/Makefile#L612